src/main/java/com/example/application/backend/entities/IdeaEntity.java

```diff
@@ -7,10 +7,10 @@
 /**
```

Comment:
In dieser Klasse wurde nichts geändert, dann kann der Header wieder zurückgesetzt werden.

Reply:
Die Kacke ist, das IntelliJ manche Änderungen irgendwie nicht geändert hat..